scripts/run_zap.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -o nounset errexit pipefail
+set -e
```

Comment:
This shouldn't be needed. `errexit` on line 3 above is the equivalent of `set -e`.

Reply:
From my testing it didn't work when the bash job run in the docker container failed, I can show you if you have a minute and see what you think.

Reply:
Scratch that you're absolutely right.
Updating those statements to the following seems to do the trick.
```bash
set -o nounset
set -o errexit
set -o pipefail
```